src/mtx/commands/uplink/getEnergyExportDayPrevious.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * Uplink command to get active energy for a previous day by 4 tariffs (`T1`-`T4`).
+ *
```

Comment:
`The corresponding downlink command: ...`

and in getEnergyDayPrevious too

Reply:
fixed